src/pages/EditStudent.tsx

```diff
@@ -76,6 +76,7 @@ const EditStudent = () => {
     setIsLoading(true);
     const stagesLength = Object.keys(STAGES).length / 2;
     const newStage = stage + 1;
+    let _studentName = studentName!.trim();
```

Comment:
Don't use Null Assertion here 
if the student name is null this throws TypeError: Cannot read properties of undefined